Swerve/SwerveTemplates/VisionBaseSwerve.java

```diff
@@ -62,6 +70,24 @@ public void updateVisionPoses() {
         }
     }
 
+    /**
+     * Method that will return an adjusted vector to nudge the robot closer to a game piece.
+     * The input should be relative to the robot, aka a Transform2d input with x = +1, y = -1, rot = pi 
+     * should move the robot 1 unit forward, 1 unit left, perform a 180 degree rotation
+     * @param driverInput a Transform2d of the driver-desired movement of the robot, relative to the robot
+     * @return an adjusted vector that will nudge the robot closer to the detected object
+     */
+    public Transform2d addVisionMovementAdjustment(Transform2d driverInput) {
```

Comment:
What units are you working with? Looks fine assuming that units are correct

Reply:
rotation in radians

rest of stuff in meters